docs: ECML2026 Railway Competition. (#24)
flatland-association/flatland-book: challenges/ecml2026.md

ECML 2026
=========

The **[ECML 2026 Challenge](https://www.aicrowd.com/challenges/flatland-3)** is the newest competition around the **Flat**land environment.

<!-- ![Flatland](../assets/images/flatland_wide.png) -->

> - Follow the [starterkit](https://github.com/flatland-association/flatland-benchmarks-f3-starterkit) to make your first submission.

Author: ```suggestion
- Follow the [starterkit](https://github.com/flatland-association/ecml2026-starterkit) to make your first submission.
```

- Read about the [evaluations metrics](ecml2026/eval) of this edition.
- Read about the [level configurations](ecml2026/levelconfig) of this edition.

⏱ Timeline
--------

* Competition start: May 4th, 2026
* Submission closure: June 8th, 2026 (AoE)
* Winner announcement: June 15th, 2026

⭐ Supported Flatland Versions
-----------------------------

> You must use Flatland version [4.2.5](https://github.com/flatland-association/flatland-rl/releases/tag/v4.2.5) (forthcoming).

Author: ```suggestion
You must use Flatland version [4.2.5](https://github.com/flatland-association/flatland-rl/releases/tag/v4.2.5).
```

ECML 2026
=========

The **[ECML 2026 Challenge](https://www.aicrowd.com/challenges/flatland-3)** is the newest competition around the **Flat**land environment.

<!-- ![Flatland](../assets/images/flatland_wide.png) -->

- Follow the [starterkit](https://github.com/flatland-association/flatland-benchmarks-f3-starterkit) to make your first submission.
- Read about the [evaluations metrics](ecml2026/eval) of this edition.
- Read about the [level configurations](ecml2026/levelconfig) of this edition.

⏱ Timeline
--------

* Competition start: May 4th, 2026

> * Sumission closure:June 8th, 2026 (AoE)

Author: ```suggestion
* Submission closure: June 8th, 2026 (AoE)
```

> * Submission closure: June 8th, 2026 (AoE)

Author: to be implemented https://github.com/flatland-association/flatland-benchmarks/issues/571

* Winner announcement: June 15th, 2026

⛽ Time and Resource limits
---

The agents have to act within **time limits**:

- You are allowed up to 30 minutes per scenario.

> - The full evaluation must finish in 4 hours.

Author: to be implemented: https://github.com/flatland-association/flatland-benchmarks/issues/583

The agents are evaluated in a container with **resource limits**

- 4 CPU cores
- 15 GB of main memory.

We do not provide GPUs.

📪 Daily Submission Limits and Submission Closure.
---
You can submit up to 2 times per day.

⭐ Supported Flatland Versions
-----------------------------

> You must use Flatland version [4.2.5](https://github.com/flatland-association/flatland-rl/releases/tag/v4.2.5) (forthcoming).

Author: TODO release with https://github.com/flatland-association/flatland-rl/pull/397/changes
